configFunctions.f90

```diff
@@ -91,21 +101,24 @@ subroutine findReactionsWithProductOrReactant( r, chs, arrayLen )
     return
   end subroutine findReactionsWithProductOrReactant
 
+  ! ----------------------------------------------------------------- !
```

Comment:
Up to you - if you want to do that, and it will help you or other users to understand what the module or its contents do, then I'm all for it :)